<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RequestBuilder</class>
 <widget class="QWidget" name="RequestBuilder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>796</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="6">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>URL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leUrl">
       <property name="text">
        <string>http://</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="cbMethod">
     <item>
      <property name="text">
       <string>GET</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>POST</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PUT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DELETE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HEAD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PATCH</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OPTIONS</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pbSubmit">
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pbGet">
     <property name="text">
      <string>GET</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="pbPost">
     <property name="text">
      <string>POST</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="pbPut">
     <property name="text">
      <string>PUT</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="6">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="contentToSendTab">
      <attribute name="title">
       <string>Content to send</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="2">
        <widget class="QPushButton" name="pbContentBase64">
         <property name="text">
          <string>Base64</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Content Options:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Content Type:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="pbContentParameter">
         <property name="text">
          <string>Parameter body</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="6">
        <widget class="QPushButton" name="pbFormatJson">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Format JSON</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="5">
        <widget class="QLineEdit" name="leContentType">
         <property name="text">
          <string>application/json</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="7">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QGroupBox" name="gbContentFile">
           <property name="styleSheet">
            <string notr="true">border:0;</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QRadioButton" name="rbContent">
              <property name="text">
               <string>Content</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbFile">
              <property name="text">
               <string>File</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leFilePath">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbBrowseFile">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0" colspan="7">
        <widget class="QPlainTextEdit" name="pteContent"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="headersTab">
      <attribute name="title">
       <string>Headers</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0" colspan="2">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leNameHeaders"/>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Value:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leValueHeaders"/>
         </item>
         <item>
          <widget class="QPushButton" name="pbAddHeaders">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pbDeleteHeaders">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" rowspan="2">
        <widget class="QTableWidget" name="tableHeaders">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Value</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="parametersTab">
      <attribute name="title">
       <string>Parameters</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="1">
        <widget class="QPushButton" name="pbDeleteParameters">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>461</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leNameParameters"/>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Value:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leValueParameters"/>
         </item>
         <item>
          <widget class="QPushButton" name="pbAddParameters">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0" rowspan="2">
        <widget class="QTableWidget" name="tableParameters">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Value</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
